transaction/src/manifest/lexer.rs

```diff
@@ -106,11 +63,22 @@ impl Lexer {
         self.text
             .get(self.current.full_index)
             .cloned()
-            .ok_or(LexerError::UnexpectedEof)
+            .ok_or(LexerError {
+                error_kind: LexerErrorKind::UnexpectedEof,
+                span: Span {
+                    start: self.current,
+                    end: Position {
```

Comment:
Good point. However, in case of EOF the `current.full_index` equals text length, which is technically not valid position in the text - but I would like to keep like this to not introduce more changes. 
Though I agree with you, that it is more clean to return span of length 0 and handle that internally - will do that :)